src/LanguageSwitcher.js

```diff
@@ -1,29 +1,46 @@
 import React from 'react';
+import styled from '@emotion/styled';
```

Comment:
we can probably remove `styled`, as `LanguageLink` doesn't have any actual styles :)